src/store/getters/state-getters.ts

```diff
@@ -94,6 +99,11 @@ export const isNamedBusiness = (state: any): boolean => {
   return !!state.stateModel.nameRequest.nrNumber
 }
 
+/** The NR number of a name request. */
```

Comment:
It's the whole NR object, not just the number.

Reply:
Fixed